api/v1alpha1/spinappexecutor_types.go

```diff
@@ -56,6 +59,11 @@ type SpinAppExecutorStatus struct {
 	// Important: Run "make" to regenerate code after modifying this file
 }
 
+// OtelConfig is the supported environment variables for OpenTelemetry
+type OtelConfig struct {
+	Endpoint string `json:"endpoint"`
+}
+
```

Comment:
```suggestion
type OtelConfig struct {
	// Sets OTEL_EXPORTER_OTLP_ENDPOINT
	OtelExporterOtlpEndpoint string `json:"otel_exporter_otlp_endpoint"`
}

```

I'd personally like to see it look like this. The stutter of the otel is annoying, but I prefer the clarity of knowing exactly which variable you're configuring

Reply:
I don't think that _feels_ particularly idiomatic (and is a lot of bonus typing):

```yaml
otel:
  otel_exporter_otlp_endpoint: "mysvc.com"
```